ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityNumericAnswer.component.ts

```diff
@@ -25,15 +25,26 @@ import * as _ from 'underscore';
         }
     `]
 })
-export class ActivityNumericAnswer implements OnInit, OnChanges {
+export class ActivityNumericAnswer implements OnInit, OnChanges, OnDestroy {
     @Input() block: ActivityNumericQuestionBlock;
     @Input() placeholder: string;
     @Input() readonly: boolean;
     @Output() valueChanged: EventEmitter<number> = new EventEmitter();
     public numericField: FormControl;
+    private subs: Subscription;
 
     public ngOnInit(): void {
         this.initForm();
+        this.subs = this.numericField.valueChanges.subscribe(enteredValue => {
+            const intValue = parseInt(enteredValue, 10);
+            const answer = isNaN(intValue) ? null : intValue;
+            this.block.answer = answer;
+            this.valueChanged.emit(answer);
+        });
+    }
```

Comment:
This is basically it. If not a number we emit null. If we get a number we emit it. Anything out of range will be caught by validation that get executed elsewhere, so we don't have to bother with that.